min-io/templates/service.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Release.Name }}-minio-service
+spec:
+  selector:
+    app: minio
+  ports:
+  - name: api-port
+    protocol: TCP
+    port: {{ .Values.service.apiPort | default 9000 }}  # External access port
+    targetPort: {{ .Values.service.apiTargetPort | default 9000 }}  # API port
+  - name: console-port
+    protocol: TCP
+    port: {{ .Values.service.consolePort | default 9090 }}  # Console access port
+    targetPort: {{ .Values.service.consoleTargetPort | default 9090 }}
```

Comment:
```suggestion
  - name: api-port
    protocol: TCP
    port: {{ .Values.service.apiPort | default 9000 }}  # External access port
    targetPort: {{ .Values.service.apiPort | default 9000 }}  # API port
  - name: console-port
    protocol: TCP
    port: {{ .Values.service.consolePort | default 9090 }}  # Console access port
    targetPort: {{ .Values.service.consolePort | default 9090 }}
```

Reply:
Should I create a separate field for the user to enter target port

Reply:
I wouldn't do that. You can map the same port to port and target port